src/utilities/core-utils.ts

```diff
@@ -92,6 +92,18 @@ const _sleep = (milliseconds: number, debug: boolean = false) => {
     );
 };
 
+/**
+ * Block execution for specified number of milliseconds, synchronously.
+ * @param milliseconds the delay
+ */
+const _sleepSync = (milliseconds: number) => {
+    let now = Date.now(),
```

Comment:
We usually don't define multiple variables with comma syntax - can you update this to be separate? 

```JS
let now = Date.now();
let start = now;
```